app/src/componentLibrary/autocompletes/multipleAutocomplete/selectedItems/selectedItems.jsx

```diff
@@ -77,7 +77,7 @@ const SelectedItem = ({
         'mobile-disabled': mobileDisabled,
         'highlight-un-stored-item': highlightUnStoredItem && !storedOption,
       })}
-      onClick={!disabled && editable && !storedOption ? changeEditMode : null}
+      onClick={!disabled && editable ? changeEditMode : null}
```

Comment:
I am not sure, that it can be fixed like this.
The reason is the following https://jiraeu.epam.com/browse/EPMRPP-78271 read the comments in this ticket.
In case of any questions, I suggest you ask our BA.

Reply:
Scenario 4. Edit value in the tag
<html>
<body>
<!--StartFragment--><div class="table-wrap" style="margin: 0px; padding: 0px; color: rgb(23, 43, 77); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Oxygen, Ubuntu, &quot;Fira Sans&quot;, &quot;Droid Sans&quot;, &quot;Helvetica Neue&quot;, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Given | 'Create Notification'/'Edit Notification'/Duplicate Notification' modal window is open At least one tag is added to 'Recipients'/'Launch names' field
-- | --
When | I click on the tag
Then | tag transforms to plain text in edit modetext remains on the tag place

</div><p style="margin: 10px 0px 0px; padding: 0px; color: rgb(23, 43, 77); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Oxygen, Ubuntu, &quot;Fira Sans&quot;, &quot;Droid Sans&quot;, &quot;Helvetica Neue&quot;, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"> </p><!--EndFragment-->
</body>
</html>
There are no additional conditions, such as "stored".